tests/drivers/build_all/sensor/sensors_trigger_own.conf

```diff
@@ -44,5 +44,5 @@ CONFIG_STTS751_TRIGGER_OWN_THREAD=y
 CONFIG_SX9500_TRIGGER_OWN_THREAD=y
 CONFIG_TMP007_TRIGGER_OWN_THREAD=y
 CONFIG_VCNL4040_TRIGGER_OWN_THREAD=y
-CONFIG_WSEN_HIDS_TRIGGER_OWN_THREAD=y
-CONFIG_WSEN_TIDS_TRIGGER_OWN_THREAD=y
+CONFIG_BMI08X_ACCEL_TRIGGER_OWN_THREAD=y
+CONFIG_BMI08X_GYRO_TRIGGER_OWN_THREAD=y
```

Comment:
Don't remove the WSEN drivers.

Please also add this new driver to `sensors_trigger_none.conf` and `sensors_trigger_global.conf`

Reply:
yeah, that was likely an issue I made when rebasing and fixing merge conflicts >_<